src/test/snapshot/test_snapshot_graphql.js

```diff
@@ -237,7 +238,13 @@ describe('Snapshot GraphQL test', () => {
             const voterAddress = Object.keys(x)[0]
             const validationValue = uni_sushi_bancor_voterValidation[voterAddress]
             if( validationValue !== undefined ) {
-                should.equal(x[voterAddress].balance.toFixed(2), validationValue.toFixed(2))
+                if (voterAddress === '0x61B15998893cC746B46C08FEdEE13a0d1b33bBa9') {
+                    console.log(`Calculated score ${x[voterAddress].balance.toFixed(2)}`)
+                    console.log(`Expected score ${validationValue.toFixed(2)}`)
+                    assert.fail('Snapshot score is not the same as expected => https://snapshot.org/#/officialoceandao.eth/proposal/QmQgfxvLqz88pL3ByK6U82bxezCU8MAiCSgxtTTCoR3fWm')
+                } else {
+                    should.equal(x[voterAddress].balance.toFixed(2), validationValue.toFixed(2))
+                }
```

Comment:
Updated unit test to fail. 
One of the voting addresses is not adding up correctly.

Validate against Snapshot results @
https://snapshot.org/#/officialoceandao.eth/proposal/QmQgfxvLqz88pL3ByK6U82bxezCU8MAiCSgxtTTCoR3fWm